kerchunk/kerchunk-in-practice.ipynb

```diff
@@ -0,0 +1,2704 @@
+{
```

Comment:
<p>When do we explain that Kerchunk doesn't work with all HDF5 files? but in a box to direct people to the issue page/discussions when things don't work.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/cloud-optimized-geospatial-formats-guide/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
I actually don't have a good resource or way of describing the limitations of kerchunk with HDF5, do you know of one? Should I just put in a comment to warn people that it's a known limitation of kerchunk that it can't work for all HDF5?

Reply:
<p>there isn't a good resource, you can lightly say generally netcdf should work, but some nested data structures and datatypes don't, and when they don't go see the tickets/issues.</p>